specification/mediaservices/resource-manager/Microsoft.Media/preview/2020-02-01-preview/MediaGraphs.json

```diff
@@ -0,0 +1,866 @@
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Start Media Graph": {
+            "$ref": "examples/media-graph-start.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/stop": {
+      "post": {
+        "summary": "Stop a Media Graph",
+        "description": "Stop a Media Graph in the Media Services account.",
+        "operationId": "MediaGraphs_Stop",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Stop Media Graph": {
+            "$ref": "examples/media-graph-stop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/operationResults/{operationId}": {
+      "get": {
+        "summary": "Get the operation result",
+        "description": "Get the operation result of a Media Graph in the Media Services account.",
+        "operationId": "OperationResults_Get",
+        "responses": {
+          "202": {
+            "description": "Accepted"

```

Comment:
202 means? Shouldn't it be 200 with a schema for response body on the operation result?

Reply:
This is for location header which returns as if it completes synchronously. For our case, when it's completed we return 204, and when it's not we return 202.